src/Database/Statement/BufferedStatement.php

```diff
@@ -111,7 +112,9 @@ public function fetch($type = parent::FETCH_TYPE_NUM)
      */
     public function fetchAssoc()
     {
-        return $this->fetch(static::FETCH_TYPE_ASSOC);
+        $result = $this->fetch(static::FETCH_TYPE_ASSOC);
+
```

Comment:
You can use  ?:

Reply:
No, I mean the short ternary form: `return $result ?: [];`. Cleaner since you do not repeat it.